<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>1151</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>951</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>Ajouter un employé</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Prénom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>50</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>CIN:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cin">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_email">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>70</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_date">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>140</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>130</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Date de naissance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_numtel">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>150</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>140</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Num</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adresse">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_etat">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>230</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>210</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Etat civil</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_salaire">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>230</y>
         <width>111</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>220</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Salaire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>310</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="le_image">
       <property name="geometry">
        <rect>
         <x>698</x>
         <y>89</y>
         <width>221</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_2">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>400</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_image">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>300</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Choisir Image</string>
       </property>
      </widget>
      <widget class="QLabel" name="le_imageurl">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>340</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Mettre à jour </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>551</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-bottom-left-radius: 15px;
    border-bottom-right-radius: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    border-top-left-radius: 15px;
    border-top-right-radius: 15px;
    padding: 5px 150px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>Mettre à jour un employé</string>
      </property>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_44">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nommaj">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_emailmaj">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_datemaj">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>190</y>
         <width>111</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_46">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>180</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Date de naissance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nummaj">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>110</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_47">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Num:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adressemaj">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_48">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Adresse:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_update">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>300</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_49">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Prénom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenommaj">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_50">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>140</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Salaire:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_salairemaj">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>150</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_51">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Etat civil:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_etatmaj">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_53">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>cin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="le_cinmaj">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="le_imagemaj">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>240</y>
         <width>191</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_imagemaj">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>350</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string>Choisir Image</string>
       </property>
      </widget>
      <widget class="QLabel" name="le_imageurlmaj">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>380</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_chercher">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
       </font>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poplar Std</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_etudiants">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_emp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>511</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>151</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
       </font>
      </property>
      <property name="text">
       <string>Trier par prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>511</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
       </font>
      </property>
      <property name="text">
       <string>Créer PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier_nom">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>141</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
       </font>
      </property>
      <property name="text">
       <string>Trier par nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_trier_cin">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>260</y>
        <width>151</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
       </font>
      </property>
      <property name="text">
       <string>Trier par cin</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>291</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>450</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>440</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed Extra Bold</family>
       </font>
      </property>
      <property name="text">
       <string>CIN:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>450</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>1021</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Statistiques</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>851</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
